2020/cmd/07/main.go

```diff
@@ -22,8 +22,10 @@ func main() {
 		return
 	}
 
+	// QUESTION: is this a good way to synchronize wait for subroutines to finish? Should I use `sync.WaitGroup`?
```

Comment:
yes, `sync.WaitGroup` would be more idiomatic, also regarding the channel more idiomatic would be `chan struct{}` for such use case

Reply:
Thanks for the answers, guys 😄